.pipelines/build-pipelines.yml

```diff
@@ -113,7 +113,6 @@ steps:
 - task: notice@0
   # This will generate the NOTICE.txt for distribution
   displayName: Generate NOTICE file
-  condition: and(succeeded(), eq(variables.isReleaseBuild, 'true'))
```

Comment:
Is removal of releaseBuild restriction only for testing?

Reply:
yes

Reply:
reverted